### PR TITLE
pps-tools: install timepps.h in location specified by RFC 2783

### PR DESCRIPTION
**Pull request by MASHtm**
This is need by eg. gpsd to build with proper PPS support.
It was already submitted once in #2892 which ended in a stalemate.
Signed-off-by: Wolfgang Breyha <wbreyha@gmx.net>

Maintainer: @wigyori
Compile tested: (arm_cortex-a9_vfpv3, OpenWrt 18.06.04, Turris OS 4.0.1)
Run tested: (arm_cortex-a9_vfpv3, OpenWrt 18.06.04, Turris OS 4.0.1, tests done)

Description:
I wanted to build a proper gspd with PPS support. gpsd build process seeks timepps.h in /usr/include/sys/ instead of the outdated location in /usr/include

### CODE REVIEW

File: utils/pps-tools/Makefile [hunk 1]
```diff
@@ -33,8 +33,8 @@ endef
 TARGET_CFLAGS += -I$(STAGING_DIR)/usr/include
 
 define Build/InstallDev
-	$(INSTALL_DIR) $(1)/usr/include
-	$(CP) $(PKG_BUILD_DIR)/timepps.h $(1)/usr/include/
+	$(INSTALL_DIR) $(1)/usr/include/sys
+	$(CP) $(PKG_BUILD_DIR)/timepps.h $(1)/usr/include/sys/
```
Comment (neheb) [inline utils/pps-tools/Makefile]: I'd use INSTALL_DATA here.

**Comment by neheb** [comment]
From a quick look, passing PKG_INSTALL:=1 should install it to the proper directory. This way PKG_BUILD_DIR can be avoided.

**Comment by MASHtm** [comment]
tried PKG_INSTALL:=1 which fails because the original Makefile does not create ($DESTDIR)/usr/bin/
I think that's the reason why PKG_BUILD_DIR is used.

**Comment by neheb** [comment]
Ah OK. Makes sense.

**Comment by MASHtm** [comment]
@wigyori please accept this pull request to make it possible to compile gpsd with PPS support

**Comment by BKPepe** [comment]
Merged into `master` branch as it seems that the maintainer is no longer interested in the package anymore.

**Pull request merged**